README.md

```diff
@@ -723,10 +723,10 @@ the url to the file on Google.
 
 ## Optimized Loading of Fog
 
-Since Carrierwave doesn't know which parts of Fog you intend to use, it will just load the entire library. If you prefer to load fewer classes into your application, you need to load those parts of Fog yourself *before* loading CarrierWave in your Gemfile.  Ex:
+Since Carrierwave doesn't know which parts of Fog you intend to use, it will just load the entire library (unless you use e.g. fog-aws instead of fog proper). If you prefer to load fewer classes into your application, you need to load those parts of Fog yourself *before* loading CarrierWave in your Gemfile.  Ex:
 
 ```ruby
-gem "fog", "~> 1.27", require: "fog/aws/storage"
+gem "fog", "~> 1.27", require: "fog/google/storage"
```

Comment:
I disagree. If you’re using AWS and want to optimize the loading of Fog, you should use the `fog-aws` gem (as mentioned under Using Amazon S3).

This section (Optimized Loading of Fog) is intended for users of older versions of Fog (v1.20 - v1.27 I guess) or users using a Fog storage library that hasn’t yet been separated from Fog.